packages/components/tests/dummy/app/templates/components/pagination.hbs

```diff
@@ -23,94 +18,316 @@
+    </li>
+    <li>
+      <code class="dummy-code">Pagination::Nav::Arrow</code>
+      - used to provide "prev/next" navigation controls
     </li>
     <li>
-      a
-      <code class="dummy-code">Pagination::Nav</code>
-      component which provides navigation controls and optionally displays page numbers
+      <code class="dummy-code">Pagination::Nav::Number</code>
+      - used to provide "page number" navigation controls
+    </li>
+    <li>
+      <code class="dummy-code">Pagination::Nav::Ellipsis</code>
+      - used to display an ellipsis instead of a set of page numbers
     </li>
     <li>
-      an optional
       <code class="dummy-code">Pagination::SizeSelector</code>
-      component which allows users to change the number of items displayed per page
+      - used to allow users to change the number of items displayed per page
     </li>
   </ul>
 
-  <h4 class="dummy-h4">Pagination</h4>
-  <p class="dummy-paragraph" id="component-api-pagination">Here is the API for the main (“container”) component:</p>
-  <dl class="dummy-component-props" aria-labelledby="component-api-pagination">
-    <dt>totalItems <code>number</code> <strong class="required">required</strong></dt>
-    <dd>Pass the total number of items to be paginated. If no value is defined an error will be thrown.</dd>
+  <p class="dummy-paragraph">
+    These pagination sub-elements may be used directly if you need to cover a very specific use need that is not covered
+    by the "numbered" or "compact" paginations. In that case, refer to the "Component API" below to see details about
+    their APIs.
+  </p>
 
+  <h4 class="dummy-h4">Pagination::Numbered</h4>
+  <p class="dummy-paragraph" id="component-api-pagination-numbered">
+    Here is the API for the
+    <code class="dummy-code">Pagination::Nav</code>
+    component:
+  </p>
+  <dl class="dummy-component-props" aria-labelledby="component-api-pagination-numbered">
+    <dt>totalItems <code>number</code> <strong class="required">required</strong></dt>
+    <dd>
+      <p>Pass the total number of items to be paginated.</p>
+      <p>If no value is defined an error will be thrown.</p>
+    </dd>
     <dt>itemsPerPage <code>number</code> <strong class="required">required</strong></dt>
     <dd>
-      Pass the maximum number of items to display on each page initially. If no value is defined an error will be
-      thrown.
+      <p>Pass the maximum number of items to display on each page.</p>
+      <p>If no value is defined an error will be thrown.</p>
+    </dd>
+    <dt>pageSizes <code>array of integers</code> <strong class="required">required</strong></dt>
+    <dd>
+      <p>Set the page sizes users can select from. If no value is defined an error will be thrown.</p>
+      <p>
+        Example:
+        {{! prettier-ignore-start }}
+        {{! template-lint-disable no-unbalanced-curlies }}
+        <code>@pageSizes=\{{array 10 30 50}}</code>
```

Comment:
Btw, I wondered if it would be useful to provide these values as default page sizes which could be overridden if desired.

Reply:
Yeah, I thought the same. Good point. Can you investigate this with @cveigt + the ambassadors, and see if it's possible to come up with two reasonable defaults for:
- `@pagesizes` (available options in the `<select>/Hds::Pagination::SizeSelect`)
- `@selectedSize` (selected option)
and in that case open a PR against this one?

Reply:
[Note: this is not a blocker for the release, because this change would not be a breaking change, so we may introduce it later as a _minor_ update]